docs/install/install-from-package/install-manager.rst

```diff
@@ -183,7 +199,7 @@ issuing SQL statement directly inside the PostgreSQL container:
 .. code-block:: console
 
    $ vfolder_host_val='{"bai-m1:local": ["create-vfolder", "modify-vfolder", "delete-vfolder", "mount-in-session", "upload-file", "download-file", "invite-others", "set-user-specific-permission"]}'
-   $ docker exec -it bai-backendai-pg-active-1 psql -U postgres -d backend \
+   $ docker exec -it $(docker ps | grep backendai-pg | awk '{print $1}') psql -U postgres -d backend \
```

Comment:
Whenever possible, we should prefer the exact reference mechanism.
Here, we could use `docker compose exec -it backendai-pg psql -U postgres -d backend` to exactly refer to the target postgres container of the current compose stack, as long as the compose stack is initialized via the cwd's `docker-compose.yml`. Otherwise, we should specify `-p` and/or `-f` options explicitly to the `docker compose` command.

Reply:
There is no need to take care of the "not beautiful" container naming scheme of Docker Compose -- it's just an auto-generated one. We should use the identity of compose stack and the service name to refer the actual container, instead of their auto-generated names.